cfme/tests/intelligence/reports/test_import_export_reports_widgets.py

```diff
@@ -145,3 +136,62 @@ def test_export_report(appliance, report_file):
         report_fields=["Archived", "Autostart", "Boot Time"],
     )
     collection.export_report(report.menu_name)
+
+
+@pytest.mark.tier(3)
+@pytest.mark.parametrize("overwrite", [True, False], ids=("overwrite", "replace"))
+def test_import_duplicate_report(appliance, report, overwrite):
+    """
+    This case tests appliance behavior when a duplicate report is imported.
+
+    Polarion:
+        assignee: pvala
+        casecomponent: Reporting
+        caseimportance: medium
+        initialEstimate: 1/4h
+    """
+    collection = appliance.collections.reports
+    file_path = yaml_path("import_report")
+    collection.import_report(file_path)
+
+    view = collection.create_view(ImportExportCustomReportsView)
+    view.flash.assert_message("Imported Report: ", partial=True)
+
+    collection.import_report(file_path, overwrite=overwrite)
+
+    if overwrite:
+        view.flash.assert_message("Replaced Report: [{}]".format(report.menu_name))
+    else:
+        view.flash.assert_message(
+            "Skipping Report (already in DB): [{}]".format(report.menu_name)
+        )
+
+
+@pytest.mark.parametrize("yaml_name", ["invalid_yaml", "invalid_report"])
```

Comment:
What is the functional difference between these two parameters?

In both cases, its just an invalid file path for import.

Can we assert any behavior against the appliance here - not just that there's an `AssertionError` raised by the `import_report` method? Can we expect a specific View to be displayed? Any evm.log or production.log messages? Any flash messages on the view?

Reply:
>What is the functional difference between these two parameters?

Both parameters have different data which gives a different error message when uploaded.

> Any flash messages on the view?

Yes, both files produce a different flash message, which we can use to assert.

Reply:
Okay, gotcha. I would like to see some docblock description of the expected content (or what's _wrong_ with the content) for the yaml files given by the parameter names.

The addition of flash message assertion certainly helps make it more clear what these files are for.